src/lcd.c

```diff
@@ -137,6 +139,32 @@ void myGPIOB_Init(void){
     GPIO_Init(GPIOB, &GPIO_InitStruct);
 }
 
+void LCD_CustomHzSymbol(void){
+  char customChar[8] = {
+    0b10001,
+    0b11111,
+    0b10001,
+    0b01110,
+    0b00010,
+    0b00100,
+    0b01000,
+    0b01110
+  };
+  // resets the RS and R/W pins
+  LCD_SendWord(LCD_COMMAND, 0x00);
+  // Sets CG RAM address
+  LCD_SendWord(LCD_COMMAND, 0x40);
```

Comment:
nit: `0x40` hides the location of the custom character within the command. IMO it's clearer to write `LCD_SendWord(LCD_COMMAND, (0x40 | HZ_SYMBOL))`

Might even be worth turning this into a function like `LCD_WriteSymbolToCGRAM(uint8_t address, uint8_t[8] symbol)`